i18n/en-US/code.json

```diff
@@ -39,6 +39,10 @@
     "message": "Read More",
     "description": "The label used in blog post item excerpts to link to full blog posts"
   },
+  "theme.blog.sidebar.navAriaLabel": {
```

Comment:
note: this was generated by the pre-commit hook script. I'm not sure why we moved the `"theme.blog.sidebar.navAriaLabel"` key up a few spaces. Might be a bug?

Reply:
The generation of those files is unfortunately not deterministic. I had to use `--no-verify` a few times the last few days 😕

@erickzhao maybe something we could contribute upstream?